.github/workflows/build.yml

```diff
@@ -99,14 +100,14 @@ jobs:
       - name: set-version-tag-cate
         uses: bc-org/gha-update-application-version-tags@main
         with:
-          app: xcube-hub
+          app: ${{ env.APP_NAME }}
           phase: ${{ steps.deployment-phase.outputs.phase }}
           delimiter: ' '
           tag: ${{ steps.deployment-phase.outputs.tag }}
           hash: ${{ steps.get-hash.outputs.hash }}
-          working-directory: ./k8s/xcube-gen/helm
+          working-directory: ./k8s/cate/helm
```

Comment:
What does the working directory refer to? is this a sub-path within k8s-configs or which directory? Because there is no directory "k8s" containing "cate". There is however "cate" containing "helm". Does the path maybe needs to be adjusted?

Reply:
This is referring to a directory teh repository k8s-config is cloned into. Otherwise you would attempt having two git repos in the working directory (k8s-config and cate-app)  which the action will refuse.

Reply:
See here in `with` section:

```yaml
name: update-version-cate-app
    steps:
      - name: git-checkout
        uses: actions/checkout@v2
      - uses: actions/checkout@v2
        with:
          repository: bc-org/k8s-configs
          token: ${{ secrets.API_TOKEN_GITHUB }}
          path: k8s
```